omnigibson/systems/micro_particle_system.py

```diff
@@ -1594,7 +1585,7 @@ def clothify_mesh_prim(cls, mesh_prim, remesh=True, particle_distance=None):
             particle_distance = cls.particle_contact_offset * 2 / 1.5 if particle_distance is None else particle_distance
 
             # Repetitively re-mesh at lower resolution until we have a mesh that has less than MAX_CLOTH_PARTICLES vertices
-            for _ in range(3):
+            for _ in range(10):
```

Comment:
i think there should be the other changes also that allow for better remeshing, right?

Reply:
we can decide whether to include those changes into `og-develop` later